modules/tinymce/src/plugins/textpattern/main/ts/api/Settings.ts

```diff
@@ -36,8 +36,8 @@ const defaultPatterns = [
   { start: '- ', cmd: 'InsertUnorderedList' }
 ];
 
-const getPatternSet = (editorSettings): PatternSet => {
-  const patterns = Obj.get(editorSettings, 'textpattern_patterns').getOr(defaultPatterns);
+const getPatternSet = (editor: Editor): PatternSet => {
+  const patterns = editor.getParam('textpattern_patterns', defaultPatterns, 'object');
```

Comment:
I gave you the wrong type here sorry @HAFRMO! This should be `array` instead of `object`, as that's the type of `defaultPatterns` and of the actual setting.

Reply:
I really should have noticed that myself too. Woups.